src/plugins/intel_gpu/include/intel_gpu/graph/program.hpp

```diff
@@ -212,6 +212,7 @@ struct program {
     std::string get_implementation_info(const primitive_id& id) const;
     const graph_optimizer_info& get_optimizer_passes_info() const;
     void save_pass_info(std::string pass_name);
+    layout get_output_layout(const program_node& node) const;
```

Comment:
How about changing name to avoid confusion? Now we have three versions of `get_output_layout` and that is quite a lot... What about something like `get_output_layout_no_touch`?
And it would be better to leave some comment on the usage or difference.

Reply:
Vladimir and I decided not to add new methods to program class and not to display any layout information if output_layout is invalid

Reply:
Great, I think that is better.

Reply:
Could you post on the channel for this change when merged?